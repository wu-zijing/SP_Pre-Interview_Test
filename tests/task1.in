8
hello
singapore
power
this
is
zijing
from
nus