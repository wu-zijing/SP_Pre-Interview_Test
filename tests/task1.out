13

20

10

10

7

14

10

8